jolt-core/src/jolt/instruction/lte.rs

```diff
@@ -0,0 +1,134 @@
+use rand::prelude::StdRng;
```

Comment:
Interestingly, I don't see the Paper describing this instruction explicitly, however we do use assert_lte in DIVU, would love to know if we want to switch this to `virtual_assert_lte.rs`